blog/nodejs-corepack.md

```diff
@@ -0,0 +1,96 @@
+# Corepack: the Node.js' manager of package managers
+
+Corepack is the new built-in tool for preparing the required package manager for our projects. We're going to see what comes with this new package for packagers. 📦
+
+## Installation
+
+We're in track to get the new Node.js LTS v16 at the end of October, without a fancy name assigned yet, and we're getting [Corepack](https://github.com/nodejs/corepack) preinstalled in the default Node.js configuration since [v16.9.0](https://github.com/nodejs/node/releases/tag/v16.9.0). 👌
```

Comment:
> We're in track to get the new Node.js LTS v16 at the end of October, without a fancy name assigned yet

We're on track to a new Node.js version, LTS v16, at the end of October, fancy name TBD.